server/controllers/pokemons/index.ts

```diff
@@ -0,0 +1,65 @@
+import { Pokemon, PokemonClient } from 'pokenode-ts';
```

Comment:
Please replace all functions to be an arrow function.

Reply:
changed